client/src/leap/soledad/client/encdecpool.py

```diff
@@ -253,9 +193,9 @@ def _encrypt_doc(self, doc):
         secret = self._crypto.secret
         args = doc.doc_id, doc.rev, docstr, key, secret
```

Comment:
We could use a namedtuple for some of this arguments and reduce the argument list of some of the functions. But this is not really in the scope of this pull-req, I'm just talking loud.